config.json

```diff
@@ -5,6 +5,8 @@
   "tags": {
     "param": "@param",
     "section": "@section",
+    "sectionDescStart": "@descriptionStart",
+    "sectionDescEnd": "@descriptionEnd",
```

Comment:
I would be consistent with the tag and the object key to make the code easier to read